manifests/Microsoft/WSL2/Kernel/5.4.42.yaml

```diff
@@ -0,0 +1,12 @@
+Id: Microsoft.WSL2Kernel
+Version: 5.4.42
+Name: WSL2 Kernel Update
+Publisher: Microsoft Corporation
+Homepage: https://docs.microsoft.com/en-gb/windows/wsl/wsl2-kernel
+License: Copyright (C) Microsoft Corporation
+Description: The WSL 2 is a new version of the architecture in WSL that changes how Linux distributions interact with Windows. WSL 2 has the primary goals of increasing file system performance and adding full system call compatibility.
+Installers:
+  - Arch: x64
+    Url: https://wslstorestorage.blob.core.windows.net/wslblob/wsl_update_x64.msi
```

Comment:
Nice idea, I also would like to have an easier option to install the WSL 2 kernel.

But I doubt this would work over a longer time. This link is a floating link to the "latest" msi package, and soon the checksum will be invalid. Is there a static link to that specific kernel version?
That would make more sense to use here in a versioned package.

Reply:
Unfortunately I couldn't find an official static link, but from the verbage on the official page (https://docs.microsoft.com/en-gb/windows/wsl/wsl2-kernel) it sounds like future updates are planned to be distributed via windows update.

I'm hoping that this is either:

* Short term fix until the initial kernel update is in windows update
* Relatively static, with future updates delivered exclusively via windows update

Reply:
Yes, I also looked around and couldn't find something. 
BTW: MSFT is also working on `wsl --install` command to improve the first time experience, adding the Windows features. I hope that we don't have to think of the kernel in near future.

Reply:
@StefanScherer - Slightly off-topic, but while we're discussing checksum validation errors - is there a recommended remediation approach for this? (Google Beta packages suffer this for example)

Reply:
@peter-dolkens I agree, only the official sources make sense. I only can suggest to get in contact with the software vendor that a static link for each released package should be provided. 

A floating link (which is „static“ eg. user can bookmark it or docs team can use it in static docs website) is nice for some use-cases. 
But it‘s bad for automation like package managers. Vendors should provide both and every user is happy. 😊